homework-8/task-1.py

```diff
@@ -0,0 +1,6 @@
+text = input("Please input text: ")
+
+for i in range(len(text)):
+	if i % 2 == 0 and text[i] != "e":
```

Comment:
პირდაპირ მხოლოდ ლუწ ინდექსებს გადაყოლოდი შეგეძლო for i in range(0, len(text), 2)-ით